internal/internal_activity.go

```diff
@@ -175,7 +175,7 @@ func getValidatedLocalActivityOptions(ctx Context) (*executeLocalActivityParams,
 
 func validateFunctionArgs(f interface{}, args []interface{}, isWorkflow bool) error {
 	fType := reflect.TypeOf(f)
-	if fType.Kind() != reflect.Func {
+	if fType == nil || fType.Kind() != reflect.Func {
```

Comment:
Curious when will this be nil? If f is nil? In that case, should we return an error? Is nil not a valid argument?

Reply:
If you pass nil as activity in ExecuteLocalActivity or ExecuteLocalActivity.
I run into this accidentally in a unit test. I defined an dummy activity in the test that takes no parameter and returns nil. I should have call workflow.ExecuteLocalActivity(ctx, dummyActivity), but I used dummyActivity() which returns nil and did workflow.ExecuteLocalActivity(ctx, dummyActivity()). The workflow panic with nil pointer error but have no clue where the error comes from.